src/components/interactive-builder/interactive-builder.component.tsx

```diff
@@ -545,6 +562,32 @@ const InteractiveBuilder: React.FC<InteractiveBuilderProps> = ({
                                           questionIndex
                                         )}
                                       </Droppable>
+                                      <br />
+                                        <p
+                                          className={responses.find((item) => item.id === question.id)?.resolution.includes("✅")? styles.approval: styles.validationError}
+                                        >
```

Comment:
Do we really need to pitch or flag positive results? Plus in my POV, `resolution.includes("✅")` this isn't the best way of classifying and identifying validation results.